### DISCUSSION TITLE
What features are omitted from open-source version, that are Included in draw.io production version?

### DISCUSSION THREAD

**Discussion by Spectrewolf8** [jgraph/drawio#4282] [Q&A]
As the title suggests, my question is that I want to know if any skipped features are included in draw.io but not in *this open-source version(_if any_). Thank you, and my apologies if the question seems a bit dumb.

**Comment by davidjgraph** [comment]
We don't keep a written list. The reasons for exclusions aren't because we go to efforts to exclude them. They are because this is a mirror repo of the main repo and some parts are hard to push out here. We have limited resources to work on everything being available, it's a best effort.

That I can think of, missing are:

- The fast realtime code. That's done as durable object on Cloudflare. That's not easy to publish. We'll get a ton of support asking how to get it working, in particular off Cloudflare.
- The gliffy import. That's in a different repo.

Nothing else springs to mind.

**Reply by Spectrewolf8** [reply]
Thank you so much for clarifying.

**Comment by osnard** [comment]
Can you maybe share which repo the gliffy import lives in?

**Reply by davidjgraph** [reply]
No, that's not possible.
